main/src/ca/uwaterloo/flix/language/errors/ExhaustiveMatchError.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2017 Jason Mitterteiner
```

Comment:
Ensure that you add yourself to `AUTHORS.md` to get credit :) and that you agree to make your contributions available under Apache 2.